### DISCUSSION TITLE
number.victron_settings_ess_acpowersetpoint

### DISCUSSION THREAD

**Discussion by Frenzeed** [sfstar/hass-victron#156] [Ideas]
would it be possible to not have this value restricted to +200/-200 please. i can change it to what ever i please via the cerbo/color screen. but i would like to be able to set it in an automation to -5000.

thanks

**Comment by pos-ei-don** [comment]
I set it in an automation via mqtt, works perfect

**Reply by Frenzeed** [reply]
while i have more learning to do(learning MQTT). i would love to know how you have it set up.
